packages/smart-wallet/src/walletFactory.js

```diff
@@ -123,10 +125,10 @@ export const makeAssetRegistry = assetPublisher => {
  *       import('@agoric/vats/src/vat-bank').AssetDescriptor>>
  * }} AssetPublisher
  *
- * @typedef {boolean} isRevive
+ * @typedef {boolean} isNew
```

Comment:
```suggestion
 * @typedef {boolean} IsNew
```

I see that it was called `isRevive`, which I find equally surprising. Since when do we introduce new type names that begin with a lowercase letter?